### ISSUE TITLE
Restore reports for level 2+

### ISSUE THREAD

**Issue by effigies** [poldracklab/fitlins#90]

**Comment by effigies** [comment]
@adelavega You gave this a thumbs up. What would you want to see in second-level reports?

**Comment by adelavega** [comment]
I think to start out stat maps would be the most helpful. I suppose choosing a threshold is an issue though, so maybe unthresholded would be fine for now? We don't do any cluster correction right?

We could show the maps at two thresholds? 1.96 and and 2.58?

It would be nice if all the different contrast maps were plotted in with the same `cut_coords`. I usually look at axial but that's not necessarily what we need to show. 

I think since most people will probably examine the final maps more closely, we don't have to try to cover all the bases, but it would be nice to get an overview of how things went without having to open jupyter or fslview.

**Comment by effigies** [comment]
We have no thresholding, let alone cluster correction. The built-in stuff for nistats is basic FDR.

The context here, though, is that when I generate a report, I look for a contrast matrix figure, which is useful simply because there is a single matrix per analysis unit, so should correspond to a single HTML file. At the first level, there are interesting things to put in a contrast matrix figure, including weights that are not all ones and it can be expanded to match the design matrix (F tests are still ambiguous, and if anybody ever writes a model that includes them, I'm sure everything will break). At the second level, the models I'm testing with are simply auto-contrasts, which are 1x1 matrices with value `[[1]]`. It's not clear that this would be useful to display, and more what anything more complex would look like and therefore how to write something that would look right.

So basically, if I know what all there might be to display, I might be able to find an alternative to adding an obligatory red square in order to get reports at the second level. It sounds like all you want is the glass brains, so I'll need another approach. I'm going to try to reorganize the figures to separate them from the NIfTI statistical maps, and hopefully I'll be able to crawl for directories to resolve it.

Perhaps @mgxd @satra have notions of what they would want to see at higher levels, since they're working with more interesting models.

**Comment by adelavega** [comment]
I suppose there are two approaches here. One is to take the "final" level as a summary of the whole analysis. So you could use that as a place to show the entire model (whether graphically or not).

If not, then yes I think the contrast plots are a bit boring, but it is what it is. Like you said, with more interesting designs, this may not be the case.

**Comment by effigies** [comment]
Now that I think about it, what would probably be good would be a subdirectory.

```
fitlins/
    sub-01/
        <images>
    ...
    reports/
        figures/
            <top-level reporting figures>
        sub-01/
            figures/
                <subject-level reporting figures>
            run-1/
                figures/
                    <run-level reporting figures>
                sub-01_task-XYZ_run-1_report.html
            sub-01_task-XYZ_report.html
        task-XYZ_report.html
 ```

This would also enable collecting lower-level reports and link to them from the higher-level.
